homeassistant/components/livisi/translations/en.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
This file should be removed from the PR. Translations are now fully automated.

Reply:
I removed the translation.